ci/unit/required_packages.sh

```diff
@@ -24,7 +24,7 @@ pkgs="gotestsum openmpi$OPENMPI_VER                \
       python3$PY_MINOR_VER-junit_xml               \
       python3$PY_MINOR_VER-tabulate numactl        \
       libyaml-devel                                \
-      valgrind-devel patchelf"
+      capstone-devel valgrind-devel patchelf"
```

Comment:
I already added capstone package in utils/scripts/install-*.sh. In my impression, I got issues due to missing capstone package in unit tests in CI without this change.

Reply:
The utils/scripts/install-sh* files are for building in docker containers so that's to be expected.  This file is for configuring nodes for running the unit tests, there is some technical debt here that we shouldn't be installing any packages via this file but that's being worked on.  If this PR needs packages added here to run then the packages should be added to rpm deps.  That said, this should only be used when running tests so I'd be surprised if a -devel package is needed, but perhaps the devel package is pulling in the non-devel version?

Reply:
Thank you! I will revise daos.spec to add the rpm dependency for capstone.

Reply:
I would verify what it is that's actually required first.

Reply:
libpil4dfs.so requires libcapstone.so. Previously, I did not add capstone as required in daos.spec for client. capstone was not installed in unit test. I just added "Requires: capstone" in daos.spec (not committed yet).

Reply:
It looks I do need to put "capstone" to install libcapstone.so in unit test.

Reply:
Then I think it needs to go into the rpm spec file.

Reply:
https://github.com/daos-stack/daos/blob/lei/DAOS-12142/utils/rpms/daos.spec#L172-L188
capstone was already added in daos.spec before I put "capstone" back into ci/unit/required_packages.sh.